topics/admin/tutorials/cvmfs/tutorial.md

```diff
@@ -189,58 +195,39 @@ Examples of data include:
 * Mutation Annotation Format (`.maf`) files
 * SAMTools FASTA indexes (`.fai`)
 
-Now all we need to do is tell Galaxy how to find it! This tutorial assumes that you have run the tutorial in the requirements, [Galaxy Installation with Ansible]({% link topics/admin/tutorials/ansible-galaxy/tutorial.md %}). The hands-on below will use the Galaxy Project Ansible role to configure everything.
+Now all we need to do is tell Galaxy how to find it! This tutorial assumes that you have run the tutorial in the requirements, "Galaxy Installation with Ansible". The hands-on below will use the Galaxy Project Ansible role to configure everything.
```

Comment:
```suggestion
Now all we need to do is tell Galaxy how to find it! This tutorial assumes that you have run the tutorial in the requirements, [Galaxy Installation with Ansible]({% link topics/admin/tutorials/ansible-galaxy/tutorial.md %}). The hands-on below will use the Galaxy Project Ansible role to configure everything.
```

reverting this because @nsoranzo changed it a couple days ago (https://github.com/natefoo/training-material/commit/1b5ceb5d6d6a251e7b49966cedaef00fd28486a8) and linking back and forth is probably a good thing.